### ISSUE TITLE
SAGU - Residência Multiprofissional - Adicionar nova nota de avaliação para o residente (mudança de tipos de notas)

### ISSUE THREAD

**Issue by vicmagpac** [EscolaDeSaudePublica/sagu#78]
# Objetivo

**Eu como** usuária do do sistema SAGU no módulo de residência multiprofissional
**Quero** realizar o lançamento de notas para o residente em uma determinada oferta de atividade do módulo
**Para que** realizar o fechamento (apto ou inapto) do residente na oferta de atividade do módulo 

# Contexto
## O que originou
Atualmente o acompanhamento das notas dos residentes em uma oferta de atividade do módulo é composta por **Atividade Produto = 30%**, **Avaliação Discente = 30%** e **Assiduidade = 40%** (Assiduidade é gerada de forma automática baseado na frequência do residente).  A solicitação nova é baseado em **30% = Atividade produto dos módulos**, **40% = Avaliação discente (30%) + autoavaliação (10%)** e **30% = questões teórico-conceitual (20%) + participação roda (10%)** (substituindo a nota de assiduidade)

## Qual solução proposta
Exemplo: 
módulo 1 - 3 notas
Nota 1 - atividade do produto
Nota 2 - Avaliação discente
Nota 3 - Avaliação teórico conceitual + participação de roda

Atividade produto: 8.9 * 3(peso) = 26.7
Atividade discente: 10 * 4(peso) = 40.0
Atividade avaliação teórico conceitual + participação de roda: 9 * 3(peso) = 27.0

Nota: 93.7/10 = 9.37

# Observações

Necessário implementar a alteração sem impacto nas turmas anteriores. Atualmente está na turma 8.
Slide apresentado na reunião dia 19/04/2021 15h.
https://docs.google.com/presentation/d/1yiofR5OomSWNAtTHGkY5RYT_UOFoUnqP/edit#slide=id.p1

# Tempo de implementação

Em média 15 dias úteis integral no desenvolvimento após priorizado

**Comment by uira** [comment]
@victorMagalhaesPacheco Já foi feita a estimativa de esforço e tempo, meu caro?

**Comment by vicmagpac** [comment]
@uira Sim! Está no final da issue. 15 dias úteis pra alterar o modelo de avaliação do residente.
